src/stream-tcp-reassemble.c

```diff
@@ -841,9 +841,41 @@ static uint64_t GetStreamSize(TcpStream *stream)
     if (stream) {
         uint64_t size = 0;
         uint32_t cnt = 0;
+        uint64_t last_ack_abs = GetAbsLastAck(stream);
+        uint64_t last_re = 0;
+
+        SCLogNotice("stream_offset %" PRIu64, stream->sb.stream_offset);
```

Comment:
this was fixed in a later commit. But really I think we can drop this commit, its not in any way critical